index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./app.css">
+    <title>Box Model</title>
+  </head>
+  <body>
+
+    <div class="navbar">
+      <h3 class="navbar__left">Popular Github</h3>
+      <a href="#" class="navbar__right">People</a>
```

Comment:
If you group the elements that should be on the right together they will be easier to float over there and maintain reading order.